Sources/SwiftDocC/Model/DocumentationMarkup.swift

```diff
@@ -154,7 +154,7 @@ struct DocumentationMarkup {
                         // Found deprecation notice in the abstract.
                         deprecation = MarkupContainer(directive.children)
                         return
-                    } else if Self.directivesRemovedFromContent.contains(directive.name) {
+                    } else if BlockDirective.directivesRemovedFromContent.contains(directive.name) {
```

Comment:
Please also remove [this duplicate `directivesRemovedFromContent` definition](https://github.com/swiftlang/swift-docc/pull/983/files#diff-88439ca0724189b95f8ea3d93247496ae24a4bc295d52c34fbc8412c5420492fR76-R82)

Reply:
Nice catch, forgot to do that! ✅